openmindat/mindat_api.py

```diff
@@ -185,24 +185,26 @@ def get_datetime(self):
         dt_string = now.strftime("%m%d%Y%H%M%S")
         return dt_string
     
-    def get_results(self, URL, JSON_DATA, PBAR):
+    def get_results(self, URL, JSON_DATA, PBAR, VERBOSE = 2):
         url = URL        
         
         try:
             response = requests.get(url, headers=self._headers)
             new_results = response.json()['results']
             JSON_DATA["results"] += new_results
```

Comment:
It is worth noting that the all-capital-letter naming for the parameters indicates that they are constant values. Please refer to [pep8](https://peps.python.org/pep-0008/#constants) for more details.
If you want to modify the constant dictionary, please consider using this format:
```
# Create a shallow copy of the constant dictionary
json_data = JSON_DATA.copy()
```